articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Personal Growth - Why every Project Should Have Gardeners
+---
+# {{page.title}}
+
+Personal Growth - Why every Project Should Have Gardeners
+
+Everyone started small. Even the tallest and strongest tree started as a seed and over time, thanks to a lot of sunlight, rainfall and good soil it became a deep-rooted, reliable and sturdy tree. And every year in the summertime it offers shade and fruits to anyone who appreciates it.
+Very much the same also applies to us, people who are involved in project work, where we try to give our best effort to make the project a success. 
+
+We work in projects as a subject matter expert, as an experienced project manager or as a newcomer that will need support from the team to grow to a new leader. Roles and tasks are assigned based on experiences we made in previous assignments. Experiences from former projects are our soil. We are able to grow on new and demanding tasks but will fail on overwhelming tasks. 
+If our responsibility lies within our comfort zone and is business as usual, we will not reach our limits. We need challenges such as finding new solutions and new approaches to grow. Just as a plant that needs more and more space to become bigger and greater we need space and the opportunity to make our own decisions and take responsibility.
+We need managers who see the potential of the project but also required skills for staffing such a project with team members. Managers that are aware of the strengths and weaknesses of each person and if there is a realistic feasibility that person can perform well and do a great job within the project.
```

Comment:
"We need managers who" _> der Begriff Managers ist etwas überladen, hier geht es ums Staffing -> "We need staffing managers" ?